components/shapediver/parameter/ParameterSelectComponent.tsx

```diff
@@ -103,6 +107,26 @@ export default function ParameterSelectComponent(
 		return uniqueChoices;
 	}, [definition.choices]);
 
+	const inputContainer = useCallback(
+		(children: React.ReactNode) => {
+			const isValid = React.isValidElement(children);
+			return (
+				<>
+					{isValid
+						? React.cloneElement(
+								children as React.ReactElement<any>,
+								{
+									onFocus: onFocusHandler,
+									onBlur: onBlurHandler,
+								},
+							)
+						: children}
+				</>
+			);
+		},
+		[focusedElement],
```

Comment:
@LemToUp shouldn't this be `onFocusHandler`and `onBlurHandler`?

Reply:
@snabela I have some skepticism to complex dependencies (functions or objects).
I mean - a simple variable will cause fewer potential problems on comparison for memoization update than function or object.

Reply:
> @snabela I have some skepticism to complex dependencies (functions or objects). I mean - a simple variable will cause fewer potential problems on comparison for memoization update than function or object.

ok, let's discuss this next week in one of our developer meetings. I would like fully understand your concern.